recipes/mergedeep/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mergedeep" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "dd95396caea02ec43e86309c30fb15168ac0be8f3df9f9bc7d3d3e49eef48ed8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/clarketm/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+
+test:
+  imports:
+    - mergedeep
+
+about:
+  home: https://github.com/clarketm/{{ name }}
```

Comment:
Just for ease of copying and pasting, could you not use jinja template variables in the `about` section for urls.